templates/10-namespace-validation.ValidatingWebhookConfiguration.yaml.tmpl

```diff
@@ -0,0 +1,27 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata: 
+  name: sre-namespace-validation
+  annotations:
+    # Typically  managed.openshift.io/inject-cabundle-from: namespace/configmap
+    # The configmap must have the cert in PEM format in a key named service-ca.crt.
+    # Each webhook in this object with a service clientConfig will have the bundle injected.
+    "#VWC_ANNOTATION#": "#NAMESPACE#/#CABUNDLECONFIGMAP#"
+webhooks:
+  - clientConfig:
+      service:
+        namespace: "#NAMESPACE#"
+        name: "#SVCNAME#"
+        path: /namespace-validation
+    failurePolicy:
+      "Fail"
+    name: namespace-validation.managed.openshift.io
+    rules: 
+      - operations: 
+          - "UPDATE"
+        apiGroups: 
+          - ""
+        apiVersions: 
+          - "*"
+        resources: 
+          - "namespaces"
```

Comment:
Could we please have a Cluster `scope` specified here, just like we have for the group-validation webhook?

Reply:
done!

Reply:
Probably we could get away without it, but standards-wise it's better to keep the same approach.

I checked and the default scope is value "*", which could potentially cause some issues.